scripts/setup-mel-builddir

```diff
@@ -346,5 +346,35 @@ if [ -e $BUILDDIR/conf/local.conf ]; then
 fi
 
 if [ $toasterconfig -eq 1 ]; then
-    $layerdir/scripts/toaster_configuration "$BUILDDIR" "$MELDIR" "$BITBAKEDIR"
+    machine=$(echo "$@" | awk -F" " {'print $NF'})
+    cmdline_string="$MELDIR/meta-mentor/setup-environment "
+    check_layer_option=0
+    for i in "$@"; do
+       if [ "$i" = "-t" ]; then
+          continue
+       fi
+       if [ "$i" = "-l" ]; then
+          check_layer_option=1
+          cmdline_string="$cmdline_string $i"
+          continue
+       fi
+       if [ $check_layer_option -eq 1 ]; then
+            check_layer_option=0
+            i="\"$i\""
+            cmdline_string="$cmdline_string $i"
+            continue
+       fi
+       if [ "$i" = "$machine" ]; then
+           cmdline_string="$cmdline_string -b \"%BUILDDIR%\" $machine"
```

Comment:
@kergoth The build directory name will be passed by toaster. And that happens when you trigger any builds from toaster UI. So lets say if you have created a project ( consider it to be first project created in toaster ), if you trigger to build console-image, then a new build directory will be created with build-toaster-2. Similary if you create another project, then the build directory will change to build-toaster-3. Your observation is much appreciated here. So I will check again for command line build, because for UI builds we need a -b option.